components/scream/scripts/perf_analysis.py

```diff
@@ -173,7 +173,7 @@ def run_test(self, test_cmd):
 
         cmd = self.formulate_cmd(test_exe)
         results = []
-        with open("{}.perf.log".format(test_exe), "w") as fd:
+        with open("{}.perf.log".format(os.path.split(test_exe)[1].split(" ")[0]), "w") as fd:
```

Comment:
Just courious: why the second split? Shouldn't `path.split(fname)[1]` already return the file name?

Reply:
path.split splits the path from the executable and args, the second split isolates the executable from the args.